rest_framework_filters/backends.py

```diff
@@ -30,8 +30,9 @@ def get_filterset_class(view, queryset=None):
             filterset_class = original(view, queryset)
 
             # django-filter compatibility
-            if issubclass(filterset_class, FilterSet):
-                filterset_class = filterset_class.disable_subset(depth=1)
+            if filterset_class:
```

Comment:
I would exit early instead of nesting the conditions

```python
# ViewSets may not generate a filterset_class
if filterset_class is None:
    return 

# django-filter compatibility
if issubclass(filterset_class, FilterSet):
    ...
```